quiz.css

```diff
@@ -0,0 +1,30 @@
+article{
+  width: 33%;
+  border: solid lightblue 1px;
+  display: inline-block;
+}
+
+h1{
+  border: dotted lightgray 1px;
+  text-align: center;
+}
+
+.image{
+  text-align: center;
+}
+
+.specifications div:nth-of-type(n+0){
```

Comment:
To make sure its clear, the comment is on line 16.  The main point I want to make is that `.specifications div:nth-of-type(n+0)` is just like saying `.specifications div:nth-of-type(n)` which is just like saying `.specifications div.  All of those are saying style all of the `div`s nested at any level under an element with the class `.specifications` in the following way.  What you have is not necessarily wrong, it's just not necessary.  Ya dig?